tests/unit/test_pipeline_graph.py

```diff
@@ -0,0 +1,80 @@
+from easylink.pipeline_graph import PipelineGraph
+
+
+def test__create_graph(default_config, test_dir):
+    pipeline_graph = PipelineGraph(default_config)
+    assert set(pipeline_graph.nodes) == {
+        "input_data",
+        "1_step_1",
+        "2_step_2",
+        "3_step_3",
+        "4_step_4",
+        "results",
+    }
+    expected_edges = [
+        (
+            "input_data",
+            "1_step_1",
+            {
+                "files": [
+                    f"{test_dir}/input_data1/file1.csv",
+                    f"{test_dir}/input_data2/file2.csv",
+                ]
+            },
+        ),
+        ("1_step_1", "2_step_2", {"files": ["intermediate/1_step_1/result.parquet"]}),
+        ("2_step_2", "3_step_3", {"files": ["intermediate/2_step_2/result.parquet"]}),
+        ("3_step_3", "4_step_4", {"files": ["intermediate/3_step_3/result.parquet"]}),
+        ("4_step_4", "results", {"files": ["result.parquet"]}),
+    ]
+    pipeline_edges = pipeline_graph.edges(data=True)
+
+    for edge in pipeline_edges:
```

Comment:
Is there a way to easily do something like `assert pipeline_edges == expected_edges`?

Reply:
graph.edges() returns something called a `OutMultiEdgeView` which is what i was trying to work around. Apparently the docs say that you can just wrap this in `list()`. Thanks!